metadata.yaml

```diff
@@ -23,7 +23,7 @@ series:
     - kubernetes
 requires:
     grafana-source:
-        interface: grafana-dashsource
+        interface: grafana-datasource
```

Comment:
Would it be a good idea to run this through @jlounder ? If is ok with this I am.